region/crateria/east.json

```diff
@@ -2171,13 +2248,50 @@
         {
           "from": 2,
           "to": [
+            {
+              "id": 1,
+              "strats": [
+                {
+                "name": "G-Mode Morph to overload Crumble Block Plms",
+                "notable": false,
+                "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [2],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    {"or": [
+                      {"and": [
+                        "Gravity",
+                        "h_canArtificialMorphIBJ"
+                      ]},
+                      {"and": [
+                        "SpringBall",
+                        "HiJump",
+                        "Bombs"
+                      ]}
+                    ]}
+                  ],
+                  "note": "Place bombs at the far right, next to the low underwater ceiling."
+                }
+              ]
+            },
             {
               "id": 3,
               "strats": [
                 {
                   "name": "Base",
                   "notable": false,
-                  "requires": []
+                  "requires": [
+                    {"or": [
+                      "canCarefulJump",
+                      "HiJump",
+                      "canWalljump",
+                      "Gravity",
+                      "h_canFly",
```

Comment:
Gravity alone isn't helpful.

Walljump doesn't seem helpful without grav.
IBJ, HiJump cannot make it out of the water.
SpringballJumpMidAir is pretty hard and I'd give it a canTrickyJump.  Or pair it with grav.

Add `canUseFrozenEnemies`.

But I'd be ok leaving this without requirements.  There's no time pressure and you can retry right away.

Reply:
I could see inexperienced players thinking this isn't possible. and the careful jump is a _little_ tricky

Reply:
Hijump/wall jump/gravity are from the upper ledge. Its a lot less tricky to get part way up compared to getting all the way onto the ledge while bootless/suitless. I removed spring ball jump midair, because it can be done with careful jump instead